spec/duration.html

```diff
@@ -364,15 +364,28 @@ <h1>CreateDuration ( _years_, _months_, _days_, _hours_, _minutes_, _seconds_, _
     </emu-clause>
 
     <emu-clause id="sec-temporal-balanceduration" aoid="BalanceDuration">
-      <h1>BalanceDuration ( _days_, _hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_ )</h1>
+      <h1>BalanceDuration ( _days_, _hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_, _largestUnit_ )</h1>
       <emu-alg>
+        1. Assert: _largestUnit_ is one of *"days"*, *"hours"*, *"minutes"*, or *"seconds"*.
         1. Let _bt_ be ? BalanceTime(_hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_).
         1. Increment _days_ by _bt_.[[Days]].
+        1. Set _hours_ to _bt_.[[Hour]].
+        1. Set _minutes_ to _bt_.[[Minute]].
+        1. Set _seconds_ to _bt_.[[Second]].
+        1. If _largestUnit_ is *"hours"*, *"minutes"*, or *"seconds"*, then
+          1. Increment _hours_ by 24 &times; _days_.
+          1. Set _days_ to 0.
+        1. If _largestUnit_ is *"minutes"* or *"seconds"*, then
+          1. Increment _minutes_ by 60 &times; _hours_.
+          1. Set _hours_ to 0.
+        1. If _largestUnit_ is *"seconds"*, then
+          1. Increment _seconds_ by 60 &times; _minutes_.
+          1. Set _minutes_ to 0.
```

Comment:
Also put [milli,micro,nano] seconds in [0,999], right?

Reply:
That should have already been done by BalanceTime. (We don't support largest-unit for sub-second units (yet) so they have to be in [0,999] after balancing no matter what.)

Reply:
So, was the spec already doing the #388 behavior?

Reply:
Sort of. The constructor was balancing up to days instead of hours, so that's what this PR changes. #388 is talking about Duration.from() which this PR doesn't make possible either; we only get that after we merge #405. (This applies to both spec and polyfill.)